templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -0,0 +1,91 @@
+param environmentName string
```

Comment:
Thoughts on having the `containerApp` module also be the entry point in constructing the ACR & CAE?  If it already exists use it, otherwise create it?

Reply:
Same comment as before, yes, but as a separate improvement.